src/codegen/codegen_c_visitor.cpp

```diff
@@ -3063,9 +3058,8 @@ void CodegenCVisitor::print_mechanism_range_var_structure(bool print_initialiser
 
     for (auto const& [var, type]: info.neuron_global_variables) {
         auto const name = var->get_name();
-        printer->fmt_line("{}* {}{}{};",
+        printer->fmt_line("{}* __restrict__ {}{};",
```

Comment:
@alkino : I would say even get rid of this restrict. Nowadays we rely everywhere on something like `pragma ivdep` or `pragma simd`. The `restrict` part is not sufficient.

So if you prefer to remove it, I am Ok with it.

Reply:
done